HTTP/action_request.json

```diff
@@ -29,7 +29,13 @@
       },
       "json": {
         "description": "The JSON to attach as body of the request",
-        "type": "object"
+        "oneOf": [
+          {"type": "object"},
+          {
+            "type": "array",
+            "items": {"type": "object"}
+          }
+        ]
```

Comment:
I think you can simply use `"type": ["object", "array"]`: https://json-schema.org/understanding-json-schema/reference/type#type-specific-keywords

Reply:
I didn't know thanks!
However this current solution specifies the item type in the array. It doesn't allow having list of strings for instance, so I'm going to stay with that.